icons/aboutdotme.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>About.me</title><path d="M19.536 9.146c-1.373 0-2.133 1.014-2.294 2.116h4.608c-.125-1.05-.867-2.115-2.314-2.115m-2.26 3.617c.235 1.156 1.193 1.97 2.532 1.97.725 0 1.77-.27 2.384-.914l1.175 1.35c-1.064 1.11-2.653 1.426-3.74 1.426-2.64 0-4.697-1.906-4.697-4.606 0-2.535 1.894-4.62 4.57-4.62 2.585 0 4.5 1.98 4.5 4.604v.766h-6.723v.023zm-6.487 3.83v-5.69c0-.976-.435-1.536-1.338-1.536-.814 0-1.355.585-1.717 1.007v6.24h-2.35v-5.7c0-.976-.415-1.532-1.318-1.532-.813 0-1.375.586-1.717 1.006v6.24H0V7.505h2.35v1.15c.4-.463 1.302-1.26 2.71-1.26 1.247 0 2.096.526 2.477 1.59.524-.761 1.5-1.59 2.91-1.59 1.7 0 2.69 1.01 2.69 2.963v6.24h-2.353l.005-.007z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>about.me</title><path d="M11.49 16.678v-6.102c0-1.008-.447-1.686-1.55-1.686-.91 0-1.763.62-2.17 1.24v6.548H5.736v-6.102c0-1.008-.426-1.686-1.531-1.686-.911 0-1.744.639-2.171 1.24v6.548H0V7.321h2.034V8.58c.407-.562 1.628-1.492 3.062-1.492 1.394 0 2.247.698 2.557 1.705.523-.833 1.764-1.705 3.178-1.705 1.705 0 2.694.93 2.694 2.829v6.761H11.49Zm3.17-4.688c0-2.713 1.977-4.902 4.748-4.902 2.809 0 4.592 2.131 4.592 5.076v.504h-7.209c.156 1.395 1.2 2.577 2.964 2.577.931 0 1.995-.369 2.674-1.047l.93 1.338c-.95.891-2.326 1.376-3.798 1.376-2.828-.001-4.901-1.958-4.901-4.922Zm4.728-3.235c-1.724 0-2.538 1.337-2.616 2.46h5.271c-.059-1.104-.814-2.46-2.655-2.46Z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>about.me</title><path d="M11.427 16.615v-6.042c0-.997-.444-1.669-1.541-1.669-.906 0-1.754.614-2.159 1.228v6.483H5.704v-6.042c0-.997-.423-1.669-1.523-1.669-.905 0-1.734.633-2.158 1.228v6.483H0V7.351h2.023v1.247C2.428 8.04 3.642 7.12 5.068 7.12c1.386 0 2.235.69 2.543 1.688.52-.825 1.754-1.688 3.16-1.688 1.697 0 2.68.92 2.68 2.8v6.694h-2.024zM24 12.163c0-2.925-1.788-5.042-4.604-5.042-2.777 0-4.759 2.174-4.759 4.869 0 2.945 2.079 4.888 4.913 4.89 1.476 0 2.855-.482 3.807-1.368l-.932-1.328c-.68.673-1.747 1.04-2.68 1.04-1.768 0-2.815-1.174-2.971-2.56H24v-.5zm-7.245-.943c.077-1.116.893-2.444 2.622-2.444 1.845 0 2.602 1.347 2.66 2.444h-5.282z"/></svg>
```

I see [some minor issues](https://github.com/simple-icons/simple-icons/assets/35000807/88213e1c-e524-4224-b25f-3c7658703d5f)
This should fix it